DQM/Integration/python/clients/pixel_dqm_sourceclient-live_For_Timing_Scan_cfg.py

```diff
@@ -0,0 +1,182 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Also, if this is intended to run in online DQM, you will need to write an email to core DQM once this is merged and this code reaches the online DQM. We will need to properly link it, so the daemon picks it up and cmsRuns it.

If it is not intended to run in online DQM, please move this file elsewhere. Preferably outside DQM/Integration.